src/main/java/core/basesyntax/exeptions/InvalidUserDataException.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.exeptions;
```

Comment:
There is a typo in the package name `exeptions`. It should be `exceptions` to maintain consistency and adhere to standard naming conventions.